cfep-03.md

```diff
@@ -25,15 +25,36 @@ But a system/protocol to manually upload builds on conda-froge anaconda channel
 
 A protocol to manually upload builds could look something like that :
 
-- Manually build the different packages for all targeted platforms.
-- Test for each platform on different version (multiple Linux distributions, Windows 7/10, etc).
-- Someone from the @conda-forge/core (having the permissions to upload on the conda channel) use conda command line to upload the package.
-- Build is available in the conda-forge anaconda channel.
+- The user provides the recipe as a PR.
+
+- The build fails : someone from @conda-forge/core decides to "enable" the manually uploaded builds system.
+
+- People from @conda-forge/core are designated to build, test and upload the package.
+
+For Linux, the `scripts/run_docker_build.sh' should be used (does it needs to be modified ?). Building with this script will provide consistency for these packages because they will be build against the same Linux distribution.
+
+On Windows, we could use this : ContinuumIO/vagrant-images#1. (Need help here I have no expertise with Windows)
+
+On OS X, same comment as before I don't know/use OS X.
+
+We also need a way to differentiate manually uploaded packages from "main" packages. Maybe using different build label `main` and `manual`.
+
+- Test for each platform and different versions the package. The test should be the one already written in the `meta.yaml` file.
+
+For Linux, test at least Debian based distribution and Fedora/CentOS. (Any suggestions for more Linux distribution ?).
+
+For windows, test at least Windows 7 and Windows 10. (do we need Windows 7 ?)
+
+For OS X, without expertise it's hard for me to tell which version should be tested.
```

Comment:
Currently we support 10.9 a.k.a. Mavericks. Maybe testing on 10.11 a.k.a. El Capitan.

Reply:
No need to specify this in the cfep. It should be in the docs.